client/src/App.jsx

```diff
@@ -21,7 +21,14 @@ const App = props => (
       <Route exact path="/unauthorized" render={() => <UnauthorizedPage {...props} />} />
       <Route exact path="/candidate" render={(routeProps) => <CandidatePage {...props} uuid={new URLSearchParams(routeProps.location.search).get('key')} />} />
       <PrivateRoute exact path="/" pageProps={props} render={() => <OverviewPage {...props} />} />
-      <PrivateRoute exact path="/calendar" pageProps={props} render={() => <CalendarPage {...props} />} />
+      <Route exact path="/calendar"
+        render={(routeProps) =>
+          <PrivateRoute exact path="/calendar" pageProps={props} render={() =>
+            <CalendarPage {...props} id={new URLSearchParams(routeProps.location.search).get('id')} />
```

Comment:
What is this for?

Reply:
Nevermind I know what is this for :D

Reply:
I think we need to figure out a better way to do this....
If we try the test case:
1. Go to Overview Page and click on the icon to schedule interview.
2. Fill out Request Dialog and book event.
3. Notice the Request Dialog automatically opens again.

Reply:
Ohh this is surely a bug. Let me try to think of a better way to do this

Reply:
Or easy way is to redirect them to /calendar without params when they hit save, how’s that?

Reply:
Sounds like an okay fix to me... so after the success message it should redirect to /calendar (be careful with timing)